apps/antalmanac/src/routes/Error.tsx

```diff
@@ -0,0 +1,25 @@
+import { Typography, Button, Stack, Box } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+const ErrorPage = () => (
```

Comment:
```suggestion
export const ErrorPage = () => (
```
nit: prefer exporting non-default components. Although large portions of the AA codebase uses default exports, it's better practice to export (and import) components more explicitly